keycloak/keycloak-admin-ui
# Realm settings(keys): add rsa-generated provider details

## Motivation

Closes #870  ; Towards #639

Also https://issues.redhat.com/browse/APPDUX-1022


## Brief Description

Adds details/edit functionality for rsa-generated providers in keys tab of realm settings.

## Verification Steps


1. Go to Realm settings -> Keys -> Providers
2. Select "rsa-generated", populate fields in modal, and save
3. Find the new provider you just created in the list and click on it.
4. See the new "rsa-generated" details page. 
5. Edit some fields, save, and navigate elsewhere in the app.
6. Navigate back to your updated rsa-generated provider and verify that the data is saved correctly.

## Checklist:


## Additional Notes
